3-networks-dual-svpc/modules/base_env/main.tf

```diff
@@ -36,111 +36,141 @@ locals {
 
   supported_restricted_service = [
     "accessapproval.googleapis.com",
-    "adsdatahub.googleapis.com",
     "aiplatform.googleapis.com",
     "alloydb.googleapis.com",
-    "alpha-documentai.googleapis.com",
     "analyticshub.googleapis.com",
     "apigee.googleapis.com",
     "apigeeconnect.googleapis.com",
+    "apikeys.googleapis.com",
+    "apphub.googleapis.com",
     "artifactregistry.googleapis.com",
-    "assuredworkloads.googleapis.com",
     "automl.googleapis.com",
+    "backupdr.googleapis.com",
     "baremetalsolution.googleapis.com",
     "batch.googleapis.com",
+    "beyondcorp.googleapis.com",
+    "biglake.googleapis.com",
     "bigquery.googleapis.com",
     "bigquerydatapolicy.googleapis.com",
     "bigquerydatatransfer.googleapis.com",
     "bigquerymigration.googleapis.com",
-    "bigqueryreservation.googleapis.com",
     "bigtable.googleapis.com",
     "binaryauthorization.googleapis.com",
+    "blockchainnodeengine.googleapis.com",
+    "certificatemanager.googleapis.com",
     "cloud.googleapis.com",
+    "cloudaicompanion.googleapis.com",
     "cloudasset.googleapis.com",
     "cloudbuild.googleapis.com",
-    "clouddebugger.googleapis.com",
+    "cloudcontrolspartner.googleapis.com",
     "clouddeploy.googleapis.com",
     "clouderrorreporting.googleapis.com",
     "cloudfunctions.googleapis.com",
     "cloudkms.googleapis.com",
     "cloudprofiler.googleapis.com",
-    "cloudresourcemanager.googleapis.com",
```

Comment:
The perimeter is configured so that only services that are in this list can be used inside the perimeter. 

Removing `cloudresourcemanager.googleapis.com` from the list would not make usage of this service fail inside the perimeter?

same question regarding removing  `iam.googleapis.com`

Reply:
huh, I have underestimated this commit. I expected that adding the latest results from progammatically checking supported services would be a simple matter of adding the latest services to reach GA. But did not expect a bunch of services to be removed.

I did this based on the result from  `gcloud beta access-context-manager supported-services list`, however, there are many services that have contradictory info on the doc you referenced:

- retail: docs say GA, gcloud says beta
- assuredworkloads: docs say GA, gcloud says beta
- dialogflow: docs say GA, gcloud says beta
- domains: docs say GA, gcloud says beta
- gkebackup: docs say preview, gcloud says beta
- meshconfig: docs say GA, gcloud says beta
- cloudresourcemanager: docs say GA, gcloud says beta
- iam: docs say GA, gcloud says beta

You're right to call out that removing CRM and IAM would have some complications, and in general I can't explain why these sources give different answers. I think for the purpose of this PR focused on dry-run, I'll undo the commit modifying services and track that as a separate issue to modify once we understand all the discrepancies.